stable/aws-for-fluent-bit/README.md

```diff
@@ -87,7 +107,7 @@ helm delete aws-for-fluent-bit --namespace kube-system
 | `kinesis.extraOutputs` | Append extra outputs with value | `""` |
 | `elasticsearch.enabled` | Whether this plugin should be enabled or not, [details](https://docs.fluentbit.io/manual/pipeline/outputs/elasticsearch) | `true` | ✔
 | `elasticsearch.match` | The log filter | `"*"` | ✔
-| `elasticsearch.awsRegion` | The region which your Firehose delivery stream(s) is/are in. | `"us-east-1"` | ✔
+| `elasticsearch.awsRegion` | The region in which your Amazon OpenSearch Service is/are in. | `"us-east-1"` | ✔
```

Comment:
nit: The region in which your Amazon OpenSearch Service cluster is in.

Reply:
done